tokio-trace-futures/src/lib.rs

```diff
@@ -4,7 +4,7 @@ extern crate tokio;
 #[cfg_attr(test, macro_use)]
 extern crate tokio_trace;
 
-use futures::{Async, Future, Poll, Sink, StartSend, Stream};
```

Comment:
Was this import unused before this change? Removing it is fine either way, just curious.

Reply:
yeah, apparently. cargo gave the warning first time I run cargo build

Reply:
cool, thanks for fixing it!

Reply:
funny, turns out it was being required on tests?